app/points/domain/workload.js

```diff
@@ -7,7 +7,7 @@ class Workload {
           sdrsDueNext30Days, sdrConversionsLast30Days,
           paromsCompletedLast30Days, paromsDueNext30Days, custodyTiers,
           communityTiers, licenseTiers, licenseCasesLast16Weeks,
-          communityCasesLast16Weeks) {
+          communityCasesLast16Weeks, armsCommunityCases, armsLicenseCases) {
```

Comment:
So we store number of cases for display on front end?

Reply:
Yup